kod/object/item/passitem/attmod/gauntlet.kod

```diff
@@ -86,6 +86,23 @@ properties:
 
 messages:
 
+   ReqUseSomething(what = $)
+   {
+      if IsClass(what,&Gauntlet)
+      {
+         if Send(poOwner,@TryUnuseItem,#what=self)
+         {
+            propagate;
+         }
+         else
```

Comment:
You don't need "else" here, since the "then" clause always returns.  This is done a little more succinctly in shield.kod:

```
   ReqUseSomething(what = $)
   {
      if isClass(what,&Shield)
      {
         if not send(poOwner,@TryUnuseItem,#what=self)
         {
            return FALSE;
         }
      }
      
      propagate;
   }

```

Reply:
Doh! Yes, indeed.